### ISSUE TITLE
Add License?

### ISSUE THREAD

**Issue by argvrutter** [SorcererX/RoboClaw#1]
Do you intend to add a license? I'd like to use and modify this repo for my own projects without infringing on your copyrights.

**Comment by SorcererX** [comment]
Not really, it's just modified to work on linux based on the arduino code at http://downloads.ionmc.com/code/arduino.zip use it any way you see fit.
